src/llhttp/http.ts

```diff
@@ -594,7 +595,15 @@ export class HTTP {
     n('chunk_size')
```

Comment:
Perhaps, this should be `chunk_size_digit` and below should be just `chunk_size`?